packages/cli/src/commands/setup/deploy/deploy.js

```diff
@@ -50,6 +50,21 @@ export const builder = (yargs) => {
       description: 'Overwrite existing configuration',
       type: 'boolean',
     })
+    .option('database', {
+      alias: 'd',
+      choices: ['none', 'postgres', 'sqlite'],
```

Comment:
```suggestion
      choices: ['none', 'postgresql', 'sqlite'],
```

Syntax for Prisma is `postgresql`.